docs/reference/configuration/configuration.md

```diff
@@ -7,7 +7,20 @@ The Arm Mbed OS configuration system, a part of the Arm Mbed OS build tools, cus
 - The receive buffer size of a serial communication library.
 - The flash and RAM memory size of an Mbed target.
 
-The Arm Mbed OS configuration system gathers and interprets the configuration defined in the target in its [target configuration](../reference/adding-and-configuring-targets.html), all `mbed_lib.json` files and the `mbed_app.json` file. The configuration system creates a single header file, `mbed_config.h`, that contains all of the defined configuration parameters converted into C preprocessor macros. `mbed compile` places `mbed_config.h` in the build directory, and `mbed export` places it in the application root. `mbed compile` runs the Mbed configuration system before invoking the compiler, and `mbed export` runs the configuration system before creating project files.
+Here is how a default JSON file might look like:
```

Comment:
Is it better to say "sample JSON file" instead of "default JSON file"?

Reply:
Also I think a different example showing usage "name" and some other common fields might be more helpful. May be we can refer one of the mbed_lib.jsons

Reply:
I can add few more fields, like baud rate to the example.